R/deployDoc.R

```diff
@@ -1,27 +1,27 @@
-#' Deploy a Document
+#' Deploy a single document
 #'
-#' Deploys an application consisting of a single R Markdown document or other
-#' single file (such as an HTML or PDF document).
+#' @description
+#' Deploys a single R Markdown, Quarto document, or other file (e.g. `.html` or
+#' `.pdf`).
+#'
+#' When deploying an `.Rmd`, `.Qmd`, or `.html`, `deployDoc()` will attempt to
+#' automaticaly discover depedencies using [rmarkdown::find_external_resources()].
```

Comment:
spelling: dependencies. automatically.

Should we suggest `deploySite` or `bookdown::publish_book`, as well?

Reply:
We already have cross-references to `deploySite()` in see also, and I think we're gradually moving away from bookdown in favour of quarto so I _think_ we're ok, as is.